dcat/index.html

```diff
@@ -5977,14 +5977,15 @@ <h2>DCAT Profiles</h2>
 </section>
 
 <section id="security_and_privacy">
-    <h2>Security and Privacy</h2>
+    <h2>Security and Privacy Considerations</h2>
     <p>
-    The DCAT vocabulary supports the attribution of data and metadata to various participants such as resource <a href="#Property:resource_creator">creators</a>, <a href="#Property:resource_publisher">publishers</a> and other parties or agents via <a href="#qualified-forms">qualified relations</a>,
-      and as such defines terms that may be related to personal information.  In addition, it also supports the association of <a href="#Property:resource_rights">rights</a> and <a href="#Property:resource_license">licenses</a> with cataloged Resources and Distributions.
-      These rights and licenses could potentially include or reference sensitive information such as user and asset identifiers as <a data-cite="?ODRL-VOCAB#privacy-consideration">described</a> in [[!ODRL-VOCAB]].   Implementations that produce, maintain, publish or
-      consume such vocabulary terms must take steps to ensure security and privacy considerations are addressed at the application level.
+    The DCAT vocabulary supports the datasets that may contain personal or private information. In addition, the metadata expressed with DCAT may itself contain personal or private information, such as resource <a href="#Property:resource_creator">creators</a>, <a href="#Property:resource_publisher">publishers</a> and other parties or agents via <a href="#qualified-forms">qualified relations</a>.
```

Comment:
It seems to be that "the" could/should be removed here (but I'm no native speaker, so I may be wrong).
```suggestion
    The DCAT vocabulary supports datasets that may contain personal or private information. In addition, the metadata expressed with DCAT may itself contain personal or private information, such as resource <a href="#Property:resource_creator">creators</a>, <a href="#Property:resource_publisher">publishers</a> and other parties or agents via <a href="#qualified-forms">qualified relations</a>.
```

Reply:
+1

Reply:
Do you mean to accept only the changes to the first paragraph? There's a lot more that differs in my suggestion. (sorry)